mmv1/third_party/terraform/tests/resource_storage_object_acl_test.go

```diff
@@ -12,7 +12,7 @@ import (
 var tfObjectAcl, errObjectAcl = ioutil.TempFile("", "tf-gce-test")
 
 func testAclObjectName(t *testing.T) string {
-	return fmt.Sprintf("%s-%d", "tf-test-acl-object", randInt(t))
+	return fmt.Sprintf("%s-%d", "tf-test/acl/object", randInt(t))
```

Comment:
Changing this changes every single test object acl, we probably want to use a mix of different names in case names w/o `/` are affected.

Reply:
okay. Added back an update test with no slashes and removed the change to `testAclObjectName`.